### ISSUE TITLE
Products are shown on the Facebook backend but not on the Shop section of Store

### ISSUE THREAD

**Issue by jcnieves** [facebookincubator/facebook-for-woocommerce#136]
Hi,

I've run the setup several times but the products are only added to the backend of the facebook fan page, not the shop section.
The slider to create a catalog on the "Your Products" step is not showing at all. I've done the Reset under Advanced Section several times, removed the catalog from facebook and the collection but the slider is not shown when trying to run the setup again, it seems that slider will never come back.
Is there a way to bring the slider back again?
Here is my screenshot , it says I have 18 products but can't actually create a catalog there so they sync to the store because the slider is not there.
I have 1.6.5 plugin version.
Facebook id is HauteBoutique, thanks!
https://www.screencast.com/t/BofYgVDxfd

**Comment by urbanoantunes** [comment]
Same problem here :(

**Comment by tahoff** [comment]
@jcnieves If I go to https://www.facebook.com/pg/HauteBoutique/shop/ the shop has 24 products. Do you still see the issue? Sometimes it takes a little while for the products to appear in the shop section.

**Comment by urbanoantunes** [comment]
Same thing happened to me. I had to run the configuration again and now it's working.

**Comment by dmitridr** [comment]
From the comment in #134 , looks like this was resolved. 

If the slider doesn't show, it's usually because of the blacklist. If the slider does show, but the products do not appear for a while, it's due to review.

**Comment by jcnieves** [comment]
Working fine now Tahoff thanks.
Products need to have valid descriptions (short and long) skus, price, image, not duplicated. All this valid on ALL products or this will not work.

Thanks for the help!

**Comment by larryyw** [comment]
Hi I am also getting the same problem, I can see all products in the existing catalog on Facebook, but none in the shop page. Can you please help?

**Comment by Danaxyzw** [comment]
Hello!
I am from Romania. I have the same trouble with the Facebook store since two years now. I cant see anything in store section. Ive uploaded products from our website https://JucariiPentruTine.com in backed catalog but the store appears empty https://m.facebook.com/JucariiPentruTine/
If anyone can, please help me out! Much obliged!

**Comment by Kashifkhan75** [comment]
Hi 
I have connected my woocoommerce webpage to facebook. the products are showing connected at woocommerce but they are not visible at my face book shop . here is the link of my facebook https://business.facebook.com/LOOJIISHOP/ and my website link is www.loojii.com
<img width="1310" alt="Screenshot 2020-04-20 at 11 52 09 PM" src="https://user-images.githubusercontent.com/64038247/79789305-6653f280-8363-11ea-9dfa-d0514c2f064d.png">
<img width="1217" alt="Screenshot 2020-04-20 at 11 53 10 PM" src="https://user-images.githubusercontent.com/64038247/79789325-6f44c400-8363-11ea-877e-b3e8742b4e96.png">

I have attached the screen shots as well. kindly advise where is the problem. I am sick of trying it several times but couldn't found solution.
thanks Kashif
